main.go

```diff
@@ -183,41 +182,56 @@ func createRouterClock(db *bolt.DB, clock Clock) *mux.Router {
 }
 
 func InitDefaultAccounts(db *bolt.DB) {
+	newSchoolRequest := NewSchoolRequest{
+		School:    "test school",
+		FirstName: "test",
+		LastName:  "admin",
+		Email:     "test@admin.com",
+	}
+	_ = createNewSchool(db, newSchoolRequest, "123qwe")
+}
 
-	schoolId, err := FindOrCreateSchool(db, "test school", "no city", 0)
+func createNewSchool(db *bolt.DB, newSchoolRequest NewSchoolRequest, adminPassword string) error {
+
+	schoolId, err := FindOrCreateSchool(db, newSchoolRequest.School, "no city", 0)
 	if err != nil {
 		lgr.Printf("ERROR school does not exist: %v", err)
 	}
 
 	lgr.Printf("INFO test school id - %s", schoolId)
 
 	admin := UserInfo{
-		Name:        "test@admin.com",
-		Email:       "test@admin.com",
-		PasswordSha: EncodePassword("123qwe"),
-		FirstName:   "test",
-		LastName:    "admin",
+		Name:        newSchoolRequest.Email,
+		Email:       newSchoolRequest.Email,
+		PasswordSha: EncodePassword(adminPassword),
+		FirstName:   newSchoolRequest.FirstName,
+		LastName:    newSchoolRequest.LastName,
 		Role:        UserRoleAdmin,
 		SchoolId:    schoolId,
 	}
 
 	_, err = CreateSchoolAdmin(db, admin)
 	if err != nil {
 		lgr.Printf("ERROR school admin is not created: %v", err)
-		return
+		return err
 	}
 
-	admin.Role = UserRoleTeacher
```

Comment:
this is a bad idea to reuse an object to create a new record in db

Reply:
this comment is only about programming practice 

you used an object to create admin
then modified few fields to create another - this is the bad practice. you have to treat the objects as immutable and if you need to modified object you create new one